src/NServiceBus.AcceptanceTesting/EndpointConfigurationBuilder.cs

```diff
@@ -85,6 +86,23 @@ public EndpointConfigurationBuilder EndpointSetup<T>(Action<EndpointConfiguratio
                     configurationBuilderCustomization(bc, runDescriptor);
                 }).ConfigureAwait(false);
 
+                if (!configuration.SendOnly)
+                {
+                    if (configuration.AddressOfAuditQueue != null)
+                    {
+                        endpointConfiguration.AuditProcessedMessagesTo(configuration.AddressOfAuditQueue);
+                    }
+                    else if (configuration.AuditEndpoint != null)
+                    {
+                        if (!routingTable.ContainsKey(configuration.AuditEndpoint))
+                        {
+                            throw new ConfigurationErrorsException($"{configuration.AuditEndpoint} was not found in routingTable. Ensure that WithEndpoint<{configuration.AuditEndpoint}>() method is called in the test.");
+                        }
+
+                        endpointConfiguration.AuditProcessedMessagesTo(routingTable[configuration.AuditEndpoint]);
+                    }
+                }
+
```

Comment:
I don't think we need this at all, but I'll raise a dedicated PR instead of delaying this further